### PR TITLE
js examples + review findings

### PR DESCRIPTION
**Pull request by enikonemeth**

### CODE REVIEW

File: examples/search-filter/byDate/js.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
-```
-
+```javascript
+const response = await fetch(
+  "https://dev.demo.sensenet.com/OData.svc/Root/Content/IT/Document_Library?$filter=CreationDate gt datetime'2019-03-26T03:55:00'",
+  { credentials: "include" }
+);
 ```
```
Comment (enikonemeth) [inline examples/search-filter/byDate/js.md:6]: This REST request is not working :(

Comment (herflis) [reply examples/search-filter/byDate/js.md:6]: thanks for mentioning, I'll open a 🐜 issue for that

File: content/basic-concepts.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ metaTitle: "sensenet API - Basic concepts"
 metaDescription: "Basic concepts behind sensenet API"
 ---
 
-# Inroduction to sensenet REST API
+# Introduction to sensenet REST API
 
-sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
+Sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
```
Comment (herflis) [inline content/basic-concepts.md]:
```suggestion
sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
```
when it is used as the name of the product it is 'sensenet' with a lowercase 's' :)

File: content/basic-concepts/03-select-expand.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ metaDescription: "How to work with a selection of metadata and referenced conten
 
 # Select and expand fields
 
-sensenet OData API gives you full control over the amount of metadata you are working allowing you to reduce the size of data response and load the value of only those fields which are needed.
+Sensenet OData API gives you full control over the amount of metadata you are working allowing you to reduce the size of data response and load the value of only those fields which are needed.
```
Comment (herflis) [inline content/basic-concepts/03-select-expand.md]:
```suggestion
sensenet OData API gives you full control over the amount of metadata you are working allowing you to reduce the size of data response and load the value of only those fields which are needed.
```

**Pull request merged**